content/knowledge-base/integrations/vault-as-secret-store.md

```diff
@@ -153,34 +154,23 @@ kubectl -n vault-system exec -it vault-0 -- vault write auth/kubernetes/role/cro
 
 ### Install and Configure Crossplane
 
-1. Install Crossplane by:
+> Prerequisite: Plugin support will be available in Crossplane v1.13.0. In this doc, we
+> will use the release candidate version of Crossplane 1.12.0-rc.0.284.g190ab067.
```

Comment:
We plan to update this when Crossplane v1.13.0 is released?

Reply:
Yes, we will update it. I opened [C/C #3995](https://github.com/crossplane/crossplane/issues/3995) to make sure I update it after v1.13 is released.

Reply:
Is this 1.13 or 1.12? 🤔

Reply:
It's 1.12